.github/workflows/linkchecker.yml

```diff
@@ -0,0 +1,23 @@
+name: linkchecker
+
+on:
+  workflow_run:
+    workflows: ["build-and-deploy"]
+    types:
+      - completed
+
+jobs:
+  linkchecker:
+    runs-on: ubuntu-latest
+    steps:     
+      - name: Installation
+        run: |
```

Comment:
The pipe `|` symbol is not required for running single-line commands. You can directly put the command after run like this: `run: npm install broken-link-checker -g`